src/Log/AccessLogDataMap.php

```diff
@@ -422,7 +423,14 @@ public function getRequestTime(string $format): string
             case 'usec':
                 return sprintf('[%s]', round($time * 1E6));
             default:
-                return sprintf('[%s]', strftime($format, (int) $time));
+                $time = (int) $time;
+                return sprintf(
+                    '[%s]',
+                    IntlDateFormatter::formatObject(
+                        new DateTimeImmutable('@' . $time),
```

Comment:
This seems very wasteful :O

Reply:
The problem is that without strftime, the only way to get a localized time string is to use IntlDateFormatter. And the only way to do arbitrary formats with IntlDateFormatter... is to use formatObject(), which requires either a DateTimeInterface or an IntlCalender instance. 

I'm really frustrated that they chose to deprecate strftime and family because there's no real 1:1 replacement. :angry:

Reply:
With the most recent changes, it at least gets re-used (if they include the `%s` placeholder, that is...).